plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerDistributedQueries.java

```diff
@@ -19,13 +19,11 @@
 import io.trino.testing.sql.TestTable;
 import io.trino.tpch.TpchTable;
 import org.testng.annotations.AfterClass;
-import org.testng.annotations.Test;
 
 import java.util.Optional;
 
 import static io.trino.plugin.sqlserver.SqlServerQueryRunner.createSqlServerQueryRunner;
 
-@Test
```

Comment:
BTW i learned recently that class-level `@Test` implies all public methods (perhaps unless annotated, or something) are treated as tests.  Strange, right? Thanks @findepi for adding `ReportUnannotatedMethods`!

Reply:
True. Thanks @findepi for adding ReportUnannotatedMethods!